src/ObjectMakr.ts

```diff
@@ -226,15 +225,17 @@ export class ObjectMakr implements IObjectMakr {
         // For each name in the current object:
         for (let name in base) {
             if (base.hasOwnProperty(name)) {
-                this.functions[name] = new Function() as IClassFunction;
+                if (!this.functions[name]) {
+                    this.functions[name] = new Function() as IClassFunction;
```

Comment:
I changed IClassFunction so having function () {} is fine except a /* */ is needed inside the `{}` so it can pass `tslint`otherwise an empty block error is thrown